packages/terra-site/theming-plugin.js

```diff
@@ -33,6 +34,6 @@ module.exports = postcss.plugin('theming-plugin', () => {
       }
     });
 
-    return fs.writeFileSync('themeable-variables.json', JSON.stringify(sortHash(variables)), 'utf8');
+    return fs.writeFileSync(path.resolve(__dirname, 'themeable-variables.json'), JSON.stringify(sortHash(variables)), 'utf8');
```

Comment:
Could we pretty print the json?
`JSON.stringify(sortHash(variables)), 'utf8', 2)`

Reply:
Yeah, good idea. That will make diffs of this file easier to reason about.

Reply:
Updated: https://github.com/cerner/terra-core/pull/811/commits/205672d5bd4d59163e3899587fee1cfded292e07